prisma/schema.prisma

```diff
@@ -115,10 +115,16 @@ model WebhookOutcomeStatus {
 }
 
 model WebhookSource {
-  id   String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  name String
+  id                String    @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  providerWebhookId String?   @unique
+  secret            String?
+  issueProviderId   String    @db.Uuid
+  createdAt         DateTime  @default(now())
+  updatedAt         DateTime  @default(now())
+  deletedAt         DateTime?
```

Comment:
This model needs to include the project relationship, the project the webhook is connected to

Reply:
done